source/slang/slang-ir-loop-inversion.cpp

```diff
@@ -118,19 +118,22 @@ static IRParam* duplicateToParamWithDecorations(IRBuilder& builder, IRCloneEnv&
 // s: ...1 goto c2
 // c2: if x then goto e2 else goto l (merge at b)
 // e2: goto b
-// l: loop break=b next=d
+// l: loop break=b2 next=d
 // d: goto c1:
 // c1: if x then goto e1 else goto e3 (merge at e3)
 // e3: goto d
-// e1: goto b
+// e1: goto b2
+// b2: goto b
 // b: ...2
 //
 // s is the Start block
 // c1, c2 are the Condition blocks
 // e1, e2, e3 are the critical Edge breakers
-// l is the Loop entering block
-// d is the loop boDy
-// b is the Break block
+// l is the loop entering block
+// d is the loop body
+// b is the merge point for the outer condition
+// b2 is the new break block for the loop
+//
```

Comment:
Is it possible to update the comment on the function to reflect the changes with e4, please?